scripts/target/mame/tiny.lua

```diff
@@ -18,6 +18,8 @@
 
 CPUS["Z80"] = true
 CPUS["M680X0"] = true
+CPUS["MCS51"] = true
```

Comment:
We essentially have to whitelist every game we add on the platform to MAME tiny?

Reply:
Yes, this was the main reason for not using mametiny, it takes quite a lot of time to search in the source code what CPU and BUS are used for a ROM.